projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -149,8 +149,14 @@ build-ami-ubuntu-2004: setup-ami-share deps-ami setup-packer-configs-ami
 release-ami-ubuntu-2004: MAKEFLAGS=
 release-ami-ubuntu-2004: FULL_OUTPUT_DIR=$(MAKE_ROOT)/$(OUTPUT_DIR)/ami
 release-ami-ubuntu-2004: PACKER_TYPE_VAR_FILES=$(PACKER_AMI_VAR_FILES)
+release-ami-ubuntu-2004: export MANIFEST_OUTPUT=$(FULL_OUTPUT_DIR)/manifest.json
+release-ami-ubuntu-2004: EXPORT_AMI_BUCKET?=$(ARTIFACTS_BUCKET)
+release-ami-ubuntu-2004: AMI_S3_DST=$(EXPORT_AMI_BUCKET)/$(ARTIFACTS_UPLOAD_PATH)/ami
+release-ami-ubuntu-2004: EXPORT_AMI_DST=$(AMI_S3_DST)/$(GIT_HASH)
+release-ami-ubuntu-2004: LATEST_AMI_S3_URL=$(AMI_S3_DST)/latest/ubuntu.raw
```

Comment:
I think we want to use $(LATEST) here instead of hardcoding latest to support building these from release branches, which write to a different "latest" folder.

Reply:
I can change it to `$(LATEST)`, good call